index.html

```diff
@@ -1631,16 +1631,17 @@ <h3 id="applying">
           <code title="">related_applications</code> member
         </h3>
         <p>
+          A <dfn>related application</dfn> is an application provided by the
+          underlying application platform that has a relationship with the
```

Comment:
I'm not a fan of  "provided by" (because it might not be installed) - maybe "accessible to"?